Sources/swift-bundler/Commands/BundleCommand.swift

```diff
@@ -113,6 +122,24 @@ struct BundleCommand: AsyncCommand {
         return false
       }
 
+      if case .visionOS = platform,
+         !arguments.shouldCodesign || arguments.identity == nil
+         || arguments.provisioningProfile == nil
+      {
+        log.error(
+          "Must specify `--identity`, `--codesign` and `--provisioning-profile` when building visionOS app"
+        )
+        if arguments.identity == nil {
+          Output {
+            ""
+            Section("Tip: Listing available identities") {
+              ExampleCommand("swift bundler list-identities")
+            }
+          }.show()
+        }
+        return false
+      }
```

Comment:
Try combining this with the previous if-statement (for iOS) to reuse the code. The only thing that changes between the two is the platform name in the error message.